lib/src/mockserver/proxy.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from "express";
+import http from "http";
+import https from "https";
+
+export function proxyRequest({
+  incomingRequest,
+  response,
+  pathPrefix,
+  proxyBaseUrl
+}: {
+  incomingRequest: Request;
+  response: Response;
+  pathPrefix: string;
+  proxyBaseUrl: string;
+}) {
+  const [protocol] = proxyBaseUrl.split("://");
```

Comment:
Make sure to fail with a helpful message if we forgot to specify the protocol? Or fallback to http:// maybe?